DataConnectors/FORCEPOINT_NGFW.json

```diff
@@ -0,0 +1,116 @@
+        },
+        {
+          "title": "1.2 Install the CEF collector on the Linux machine",
+          "description": "Install the Microsoft Monitoring Agent on your Linux machine and configure the machine to listen on the necessary port and forward messages to your Azure Sentinel workspace. The CEF collector colects CEF messages on port 514 TCP.\n\n> 1. Make sure that you have Python on your machine using the following command: python –version.\n\n> 2. You must have elevated permissions (sudo) on your machine.",
+          "instructions": [
+            {
+              "parameters": {
+                "fillWith": [
+                  "WorkspaceId",
+                  "PrimaryKey"
+                ],
+                "label": "Run the following command to install and apply the CEF collector:",
+                "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+              },
+              "type": "CopyableLabel"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "2. Forward Common Event Format (CEF) logs to Syslog agent",
+      "description": "Set your security solution to send Syslog messages in CEF format to the proxy machine. Make sure you to send the logs to port 514 TCP on the machine’s IP address."
+    },
+    {
+      "title": "3. Validate connection",
+      "description": "Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python –version\n\n>2. You must have elevated permissions (sudo) on your machine",
+      "instructions": [
+        {
+          "parameters": {
+            "fillWith": [
+              "WorkspaceId"
+            ],
+            "label": "Run the following command to validate your connectivity:",
+            "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"
+          },
+          "type": "CopyableLabel"
+        }
+      ]
+    },
+    {
+      "title": "4. Secure your machine ",
+      "description": "Make sure to configure the machine's security according to your organization's security policy\n\n\n[Learn more >](https://aka.ms/SecureCEF)"
+    },
+    {
+      "title": "5. Forcepoint integration installation guide ",
+      "description": "To complete the installation of this Forcepoint product integration, follow the guide linked below.\n\n[Installation Guide >](https://support.forcepoint.com/knowledgebase#sort=relevancy)"
```

Comment:
1. This link https://support.forcepoint.com/knowledgebase#sort=relevancy points to a general table of contents, rather than specific page to configure. Can you please update with an appropriate link that goes to the right page?
2. Also have the link as a redirection URL that you can manage so that if the actual link changes, it doesn’t need a data connector update for link fix and re-deployment which takes time. Putting a fixed link creates issues with changes to links requiring re-deployment of data connector which takes a lot of time.

Reply:
We're waiting on a final link here, so this is still unresolved

Reply:
I created and added a redirection link to get this unblocked. Please share your link so that can update.